### ISSUE TITLE
will support transcribe live ?

### ISSUE THREAD

**Issue by michoael** [nyadla-sys/whisper.tflite#10]
i hope to know the plane if you will support live transcribe from mic and long file with timestamps ?

**Comment by vilassn** [comment]
Yes. We are planning to support live transcription from mic and long file with timestamps.

**Comment by aramfaghfouri** [comment]
Do you have any approximate ETA on this? thanks!

**Comment by JusticeEli** [comment]
I think it's possible to implement live transcriptions if you can manage to put the 30-second chunks of audio in some queue and run predictions for the audios in the queue as you continue recording others. There will be a lag but it will give the impression of real time.

**Comment by michoael** [comment]
You will do this soon or you will have long time?
